### PR TITLE
feat: add SPS chainId and SPSTest chianId

### PR DESCRIPTION
**Pull request by Clubswap**
Please verify the request.

The SPS and SPS-Testnet is proposed and the testnet is running on.  The SPS is incubating.

Thanks a lot.

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `SPS-TEST` more than once

**Comment by LycrusHamster** [comment]
> short name must be unique - but found `SPS-TEST` more than once

Oooops, sorry for my recklessness

**Comment by Clubswap** [comment]
> short name must be unique - but found `SPS-TEST` more than once

Sorry for my recklessness.

**Review by ligi — CHANGES_REQUESTED** [review]
the explorer URLs do not seem to work

**Comment by Clubswap** [comment]
> the explorer URLs do not seem to work

which one? 
The SPS testnet explorer https://explorer.3sps.net/ is working perfectly.
The SPS explorer might be out-of-service, cause it's incubating :)    It's fixed and running now.

**Comment by Clubswap** [comment]
Move testnet explorer from https://www.3sps.net/ to https://explorer.3sps.net/

File: _data/chains/eip155-14000.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://ssquad.games/",
+  "shortName": "SPS-Test",
+  "chainId": 14000,
+  "networkId": 14000,
+  "explorers": [
+    {
+      "name": "SPS Test Explorer",
+      "url": "https://explorer.3sps.net/",
```
Comment (ligi) [inline _data/chains/eip155-14000.json:18]:
```suggestion
      "url": "https://explorer.3sps.net",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — APPROVED** [review]

**Pull request merged**